testing/fake-dom.js

```diff
@@ -427,6 +435,7 @@ export class FakeHistory {
       throw new Error('can\'t go forward');
     }
     this.index = newIndex;
+    this.win.eventListeners.fire({type: 'popstate'});
```

Comment:
Well. It's a very optional argument. I just didn't want to explode number of private/public methods here. And `replaceState`/`pushState` are used internally for `FakeLocation`.